dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js

```diff
@@ -174,6 +174,58 @@ describe('YouTube Atom', function () {
 		cy.get(overlaySelector).should('not.exist');
 	});
 
+	it('plays in video when 2 same videos exist in page', function () {
+		// cy.intercept('**youtube**').as('callToYouTube');
+		cy.visit(
+			'/Article?url=https://www.theguardian.com/world/live/2022/mar/28/russia-ukraine-war-latest-news-zelenskiy-putin-live-updates',
+		);
+		cmpIframe().contains("It's your choice");
+		cmpIframe().find("[title='Manage my cookies']").click();
+		privacySettingsIframe().contains('Privacy settings');
+		privacySettingsIframe()
+			.find("[title='Accept all']", { timeout: 12000 })
+			.click();
+
+		// Wait for hydration
+		cy.get('[data-component=youtube-atom]')
```

Comment:
Do we need to do this for each atom?

Reply:
I wasn't sure, I just brought it because it was there in the other tests. Do you think this check is not needed?

Reply:
Oh I meant waiting to hydrating each atom. I followed your example below to add this 👍

Reply:
Nice 👍